tabpane.go

```diff
@@ -0,0 +1,235 @@
+
+package termui
+
+import (
+	"unicode/utf8"
+)
+
+type Tab struct {
+    Label   string
+	RuneLen int
+    Blocks  []Bufferer
+}
+
+func NewTab(label string) *Tab {
+    return &Tab {
+                Label:  label,
+				RuneLen: utf8.RuneCount([]byte(label)) }
```

Comment:
@marigs Looks like you haven't run `go fmt` on these files, spacing is all messed up.